minikube/docker-compose.manual-services.yml

```diff
@@ -4,17 +4,26 @@ services:
     build:
       context: ../initial-data-load
     volumes:
```

Comment:
we won't need volumes at all as we're using typescript that is being built in Dockerfile and for copying data config there is script